source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -331,6 +376,34 @@ TEST_F(NiDCPowerDriverApiTest, SetAttributeViInt64_GetAttributeViInt64ReturnsSam
   EXPECT_EQ(expected_value, get_attribute_value);
 }
 
+TEST_F(NiDCPowerDriverApiTest, ConfigureOutputFunctionAndVoltageLevel_ConfiguresSuccessfully)
```

Comment:
These only differ by attribute value. Consider using `TEST_P` with `INSTANTIATE_TEST_CASE_P`.

Reply:
@jonlugoatx Its true that they only differ by attribute value but both of these attributes are of different types like one is ViInt32 and the other is ViReal64 and also the underlying API which configures these attributes are different.